Source/WebCore/page/csp/ContentSecurityPolicy.h

```diff
@@ -87,6 +88,9 @@ enum class AllowTrustedTypePolicy : uint8_t {
     DisallowedDuplicateName,
 };
 
+using HashAlgorithmSet = uint8_t;
+using HashAlgorithmSetCollection = FixedVector<std::pair<HashAlgorithmSet, Vector<String>>>;
```

Comment:
Occurs to me that this could also use `FixedVector<String>`.

Reply:
Changed it! This also required a conversion back from FixedVector to Vector before sending the report. I created a custom one, as I don't know if that's common enough to bake in to Vector

Reply:
As a follow up we can fix that by changing `sendReportToEndpoints` to take a `std::span<const String>` instead of a `const Vector<String>&`.